aglu.md

```diff
@@ -0,0 +1,244 @@
+
+The other extreme is that of no substitution. This would be accomplished with near zero or zero logit exponents, whether in a single nest or multiple nests. This implies that either it is physically impossible or the transition costs are too high to allow switching from one land type to another. Currently, in GCAM, we use zero logit exponents in a very limited number of situations, where we do not want any substitution (e.g., we do not allow cropland to expand into desert or tundra).  However, in most situations, we employ positive logit exponents and allow economics to dictate the land allocation within a region. 
+
+Our approach is to use a nesting strategy that allows the logit exponents to reflect differences in substitutability across land categories. Figure 2 shows the nesting diagram of land with an AEZ subregion. At the top is all land, which is divided into two main types of nodes: agro-forestry land and the remaining categories of land that are not suitable for agriculture. This second category could be divided further if useful. The next node layer contains two further nodes: all agro-forestry, non-pasture land and all pasture land. The pasture land node contains two competing uses (land leaves in the code): managed pasture (that which feeds marketed livestock) and unmanaged pasture.
+
+*Figure 2: AgLU Land Nest* 
+
+![Figure 2](gcam-figs/AgLUTree.bmp)
+
+The agro-forestry (non-pasture) node contains three competing nodes: shrub and grass lands, forest lands, and croplands. Shrublands and grasslands are separated from the rest as they are both classified as unmanaged land categories and we want to control their substitutability between each other separately. Finally, the forestland node competes with the total cropland node. Within forestland, there are managed and unmanaged forest leaves,  and we have added a woody biomass option there in some regions and scenarios. Under cropland are all food and other agriculture products (e.g., corn, wheat, sugars, etc.), including biomass crops, along with an unmanaged land category called other arable land. Note that several crops are included explicitly in the CropLand node, and the grouping of “AllOtherCrops” is simply a convenience for this figure.
+
+With the specification above, we can make substitution across categories more or less difficult by choosing lower or higher logit parameters. We can also effectively combine nests by making the logit exponents the same. For example, if we assign the same logit exponent for each level, this is equivalent to putting all uses under one nest.  One implication of having equal exponents is that it would not matter in which node we placed new crop options like biomass.
```

Comment:
@kvcalvin, is it true that making the logit exponents the same is equivalent to putting them all in a single nest?  The math behind that is not obvious to me.

A quick test shows that it's *not* true using the weighted average formula for the cost of a nest (which is what we use in the energy system).  It also doesn't seem to be true when you use the average cost formula from Clarke and Edmonds, though I concede that in coding that one from memory I may have messed it up subtly.  Here are my numbers with a logit exponent of 3 and p1=1, p2=3, and p3=2:
```
python share.py
nested shares    1: 0.0270868606265  2: 0.731345236916  3: 0.241567902457
alt nested       1: 0.0273285414795  2: 0.737870619946  3: 0.234800838574
unnested shares  1: 0.0277777777778  2: 0.75  3: 0.222222222222
```

Anyhow, I think we either need to support this claim or remove it.